src/pages/GetInvolved/GetInvolved.js

```diff
@@ -1,42 +1,46 @@
 import React from "react";
-import { useSelector } from "react-redux"
+import { useSelector } from "react-redux";
 import Layout from "components/Layout/Layout";
 import { Link } from "react-router-dom";
-import './GetInvolved.scss';
+import "./GetInvolved.scss";
 
 const GetInvolved = () => {
+  const clubDataAll = useSelector((state) => state.clubs);
```

Comment:
A better practice would be to have this a `let` variable since its changing state